chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/LeftRoomTest.kt

```diff
@@ -0,0 +1,37 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.LeftRoom
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class LeftRoomTest : Spek({
+
+    describe("given a joined rooms state of two rooms") {
+        val givenState = ChatkitState(
+            joinedRoomsState = JoinedRoomsState(
+                rooms = mapOf(
+                    roomOneId to roomOne,
+                    roomTwoId to roomTwo
+                ),
+                unreadCounts = mapOf(
+                    roomOneId to 1,
+                    roomTwoId to 2
+                )
+            )
+        )
+
+        describe("when an event for leaving a room that is part of the state is received") {
```

Comment:
let's have BDD descriptions in the same terse style in all the tests as for [JoinedRoom](https://github.com/pusher/chatkit-android/pull/192#discussion_r387145321)

Reply:
👍 have updated all the other tests.